grub-core/loader/i386/linux.c

```diff
@@ -1152,19 +1152,80 @@ grub_cmd_initrd (grub_command_t cmd __attribute__ ((unused)),
   return grub_errno;
 }
 
+static grub_err_t
```

Comment:
Do you have a link to where you copied this for a reference?

Reply:
I didn't copy it from anywhere outside.
I just tried to follow the pattern of coreos/grub's other commands.